sessions/session-e.md

```diff
@@ -0,0 +1,41 @@
+# Session E: A History of Toronto's Networks
```

Comment:
Are we following a style convention with the way these sessions are titled? A good title will be helpful frame the discussion, or segue into it.